.github/workflows/release.yml

```diff
@@ -34,19 +34,28 @@ jobs:
           release: true
           signed: false
 
-      - uses: actions/download-artifact@v4
-        # download all artifacts
+      - uses: actions/checkout@v4
+
+      - name: download all action artifacs
+        uses: actions/download-artifact@v4
         with:
+          merge-multiple: true
```

Comment:
```
    # When multiple artifacts are matched, this changes the behavior of the destination directories.
    # If true, the downloaded artifacts will be in the same directory specified by path.
    # If false, the downloaded artifacts will be extracted into individual named directories within the specified path.
    # Optional. Default is 'false'
    merge-multiple:
```

Removing this option would not stop downloading the artifacts, it would just create a directory for each of them, which ist not super useful in our case, as we name the artifacts the same as the file inside the artifact.